README.md

```diff
@@ -156,15 +156,45 @@ git clone git@github.com:OpenAssetIO/OpenAssetIO
 Linux is currently the only fully tested platform. Support
 for other platforms is the subject of ongoing work.
 
+#### Quick start: Python only
+
+For use in a Python-only environment, the Python package can be
+installed from the root of the repository using
+
+```shell
+pip install .
```

Comment:
Nit: As a developer, I almost always use `pip install -e .` so I can noodle on Python code. Granted, I do that in a pure Python environment.

Perhaps this needs a little clarification for intended usage?

Reply:
:thinking:  I think people who don't know about develop installs will be wanting to install into a wider system, so the docs are appropriate, whereas people who do know about develop installs will be able to understand that they can add `-e` if they want?

Reply:
True, although when I first read it (as a developer) I wondered why the develop mode wasn't being used, almost as if it wasn't supported. Perhaps a note on that?

Reply:
:+1: added a note in 03d1a46b, is that OK?

Reply:
Thanks!